src/lib/libcmd/wclib.c

```diff
@@ -390,7 +390,6 @@ int wc_count(Wc_t *wp, Sfio_t *fd, const char *file) {
                         }
                         while (mbc(c) && ((c & WC_ERR) || (c & 7) == 0)) c = type[*cp++];
                         if (eol(c) && (cp > endbuff)) {
-                            c = WC_MB | WC_ERR;
```

Comment:
Note that I recently fixed a related bug on line 391 which had an always true condition: `((c | WC_ERR) || (c & 7) == 0)`. I now have very low confidence this code does the right thing.